Assignment_1.cu

```diff
@@ -0,0 +1,45 @@
+#include<iostream>
+using namespace std;
+int Array_Size;
+
+__global__ void Sum(float* d_in1,float* d_in2, float* d_out,int* d_array_size)
+{
+    int id = blockIdx.x * blockDim.x + threadIdx.x;
+    if(id < *d_array_size)
+     d_out[i] = d1_in[i] + d2_in[i];    
```

Comment:
error: identifier "i" is undefined
error: identifier "d1_in" is undefined
error: identifier "d2_in" is undefined